ext/spl/spl_heap.c

```diff
@@ -1192,7 +1192,7 @@ static const zend_function_entry spl_funcs_SplHeap[] = {
 	SPL_ME(SplHeap, valid,                 arginfo_splheap_void, ZEND_ACC_PUBLIC)
 	SPL_ME(SplHeap, recoverFromCorruption, arginfo_splheap_void, ZEND_ACC_PUBLIC)
 	SPL_ME(SplHeap, isCorrupted,           arginfo_splheap_void, ZEND_ACC_PUBLIC)
-	ZEND_FENTRY(compare, NULL, NULL, ZEND_ACC_PROTECTED|ZEND_ACC_ABSTRACT)
+	ZEND_FENTRY(compare, NULL, arginfo_heap_compare, ZEND_ACC_PROTECTED|ZEND_ACC_ABSTRACT)
```

Comment:
This is a BC, isn't? :confused:

Reply:
Adding arginfo is BC

Reply:
BC breaks go to PHP 8 or is it something that PHP 7.4 can handle as well?

Reply:
@petk BC goes only in PHP 8. See #3443 for example, as well [#74035](https://bugs.php.net/bug.php?id=74035) and [#75958](https://bugs.php.net/bug.php?id=75958)